flytekit/clis/sdk_in_container/run.py

```diff
@@ -656,7 +698,7 @@ class RemoteEntityGroup(click.RichGroup):
     WORKFLOW_COMMAND = "remote-workflow"
     TASK_COMMAND = "remote-task"
 
-    def __init__(self, command_name: str):
+    def __init__(self, command_name: str, h: str):
```

Comment:
Do we use `h` anywhere?

Reply:
thanks for pointing out, should use h instead, fixed